assignment/scripts/3-music-collection.js

```diff
@@ -1 +1,104 @@
 console.log('***** Music Collection *****')
+
+let collection = [];
+
+function addToCollection(title, artist, yearPublished, tracks) {
+    // console.log('this is this:', this);
+    this.title = title,
+        this.artist = artist,
+        this.yearPublished = yearPublished;
+    this.tracks = tracks;
+    collection.push(this);
+    // console.log('this is this:', this);
+    return this;
+}
+
+let album1 = new addToCollection('BB U OK?', 'San Holo', 2021,
+    [{ name: 'Find Your Way:', duration: '3:45' },
+    { name: 'Make This Moment Last:', duration: '3:16' },
+    { name: 'My Fault:', duration: '3:17' }]
+);
+let album2 = new addToCollection('Recovery', 'Eminem', 2010,
```

Comment:
Typically, there are two ways you might see people indent this.

Start indenting with the parentheses:

```js
let album2 = new addToCollection(
  'Recovery', 
  'Eminem', 
  2010,
  [
    { name: 'No Love:', duration: '5:00' },
    { name: 'Love The Way You Lie:', duration: '4:23' },
    { name: 'Not Afraid:', duration: '4:08' }
  ]
);
```

Or, start indenting with the opening bracket of the array:

```js
let album2 = new addToCollection('Recovery', 'Eminem', 2010, [
  { name: 'No Love:', duration: '5:00' },
  { name: 'Love The Way You Lie:', duration: '4:23' },
  { name: 'Not Afraid:', duration: '4:08' }
]);
```

The consistent pattern here is that brackets/braces/parens either open and close on the same line, OR the opener is the last thing on a line and then the closer is the first thing on a later line, with everything in between indented.

That said, this is just an FYI. Your approach is clean and used consistently, which I appreciate even if it is a bit unorthodox.